internal/pkg/flags/flags.go

```diff
@@ -55,7 +55,6 @@ const (
 	DefaultWatcherResourcesPath                                         = "./skr-webhook"
 	DefaultWatcherResourceLimitsCPU                                     = "0.1"
 	DefaultWatcherResourceLimitsMemory                                  = "200Mi"
-	DefaultDropStoredVersion                                            = "v1alpha1"
 	DefaultDropCrdStoredVersionMap                                      = "Manifest:v1beta1,Watcher:v1beta1,ModuleTemplate:v1beta1,Kyma:v1beta1"
 	DefaultMetricsCleanupIntervalInMinutes                              = 15
 	DefaultLeaderElectionLeaseDuration                                  = 180 * time.Second
```

Comment:
Please also remove the DropStoredVersion [here](https://github.com/kyma-project/lifecycle-manager/blob/76249ad8d6e2cbaa8f9d2388a5326683e74116bf/internal/pkg/flags/flags.go#L280)